docs/developer-guide/etendo-classic/how-to-guides/How_to_Create_a_Table.md

```diff
@@ -0,0 +1,155 @@
+
+Save this record then press the _Create columns from DB_ button to create columns within the Column tab automatically.
+
+![](../../../assets/developer-guide/etendo-classic/how-to-guides/How_to_create_a_Table-2.png)
+
+Once the creation process has finished, you will be informed of the number of columns that have been added to this table.
+
+![](../../../assets/developer-guide/etendo-classic/how-to-guides/How_to_create_a_Table-3.png)
+
+Switch to _Column_ tab to see all the columns (for more information, see the AD*Column table description) that were created according to their definition within the database. You can now additionally alter the properties of each column. Each column is assigned a reference (which defines the data type) depending on its name and its data type. Run \_Synchronize Terminology* process (`Application Dictionary` > `Synchronize Terminology`). Two tricky details before you run it:
+
+- Check you have defined dependencies from your module to core and any other needed module. Synchronize Terminology will link your columns to Elements in your module or in modules it depends on. If you don't declare the dependency to core, the system will create new elements for standard columns like IsActive, CreatedBy, etc.
+- Do not set the Elements linked to your columns, let the Synchronize terminology to do the job. This way, the process will find the appropiate elements and will set the proper column names for the standard columns (IsActive, CreatedBy, etc.).
+
+![](../../../assets/developer-guide/etendo-classic/how-to-guides/How_to_create_a_Table-4.png)
+
+This process tries to find an existing application element (within the currently developed module) and thus its label, help and description and if one is not found, a new one is created. This enables a centralized translation of the application/module.
+
+![](../../../assets/developer-guide/etendo-classic/how-to-guides/How_to_create_a_Table-5.png)
+
+Each table must have at least one column marked as an identifier. The actual values of identifier columns later get concatenated to be shown to the user as a representation of a particular record (see the link to the Sales Order within the Sales Invoice window). These identifiers will also be used to construct dropdown lists of records of that particular table. By default all columns with column name _Name_ are set as an identifier. In case there is no column with this _Name_ , no identifier is set and needs to be done so manually or compilation will fail.
+
+The name is used by the [Data Access Layer](../../../developer-guide/etendo-classic/concepts/Data_Access_Layer.md) and in [REST webservices](../../../developer-guide/etendo-classic/concepts/XML_REST_Web_Services.md). For specific columns (audit info, client/organization, active) it is important to be precise in the naming. See [here](../../../developer-guide/etendo-classic/concepts/) for more information.
```

Comment:
@mapenglishservices @Tomas-Vesco The `../concepts/` link is going to be broken. Where should it be directed?

```suggestion
The name is used by the [Data Access Layer](../../../developer-guide/etendo-classic/concepts/Data_Access_Layer.md) and in [REST webservices](../../../developer-guide/etendo-classic/concepts/XML_REST_Web_Services.md). For specific columns (audit info, client/organization, active) it is important to be precise in the naming. See [here](../concepts/) for more information.
```